src/main/java/com/maplexpbar/MapleXPBarConfig.java

```diff
@@ -38,9 +38,17 @@ default Skill skill()
 	)
 	default boolean displayHealthAndPrayer() { return false; }
 
-	@Alpha
 	@ConfigItem(
 			position = 3,
```

Comment:
I put it here to be next to the other display modifiers, but before the color selections. Could really go anywhere though.